pallets/parallel-xcm/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Parallel Finance Developer.
+// This file is part of Parallel Finance.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Common XCM Helper pallet
+//!
+//! ## Overview
+//! This pallet should be in charge of everything XCM related including callbacks and sending XCM calls.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[frame_support::pallet]
+pub mod pallet {
+    use super::*;
+
+    #[pallet::config]
+    pub trait Config: frame_system::Config {
+        type XCM: ParallelXCM;
+    }
+
```

Comment:
The `parallel-xcm` exposes the `ParallelXCM` trait. The code you currently have would be for the other pallets that uses this one.